codes/2206_py3_sjskgks.py

```diff
@@ -0,0 +1,37 @@
+import collections
+import sys
+input = sys.stdin.readline
+
+def bfs(graph):
+    visit = [[0 for _ in range(col)] for __ in range(row)] # 부순적 없이 방문하는 경우
+    crush_visit = [[0 for _ in range(col)] for __ in range(row)] # 한번 부수면서 or 부순적 있이 방문하는 경우
+    q = collections.deque([[0,0,1,0]]) # row, col, cost, crush
+    visit[0][0]=1 # row, col
+    while q:
+        cur_r, cur_c, cur_cost, crush_num = q.popleft()
+        if cur_r==row-1 and cur_c==col-1:
+                return cur_cost
+        for next_idx in [[cur_r-1,cur_c],[cur_r+1,cur_c],[cur_r,cur_c-1],[cur_r,cur_c+1]]:
+            next_row, next_col = next_idx
+            if next_row<row and -1<next_row and next_col<col and -1<next_col:

```

Comment:
이러면 아래에 indent가 전부 하나씩 추가되므로
```suggestion
            if next_row >= row or 0 > next_row or next_col >= col or 0 > next_col:
                continue
```
로 바꾸고 아래 인덴트를 하나씩 줄이는 것이 좋습니다.